app/src/views/DrefApplicationExport/PgaExport/index.tsx

```diff
@@ -0,0 +1,248 @@
+                                            <br />
+                                            <i>{strings.nationalSocietyDescriptionEleven}</i>
+                                        </li>
+                                    </ol>
+                                </li>
+                            </ol>
+                        </li>
+                        <li>
+                            {strings.nationalSocietyDescriptionTwelve}
+                            <ol type="i">
+                                <li>
+                                    <i>{strings.nationalSocietyDescriptionThirteen}</i>
+                                </li>
+                            </ol>
+                        </li>
+                        <li>
+                            {strings.nationalSocietyDescriptionFourteen}
+                            <ol type="i">
+                                <li>
+                                    <i>{strings.nationalSocietyDescriptionThirteen}</i>
+                                </li>
+                                <li>
+                                    <i>{strings.nationalSocietyDescriptionFifteen}</i>
+                                </li>
+                            </ol>
+                        </li>
+                    </ol>
+                </DescriptionText>
+            </Container>
+            <Container>
+                <DescriptionText>
+                    {strings.nationalSocietyDescriptionSixteen}
+                </DescriptionText>
+            </Container>
+            <Heading level={4}>
+                {strings.nationalSocietyBankDetails}
+            </Heading>
+            <Container>
+                <DescriptionText className={styles.tableDescription}>
+                    {strings.nationalSocietyBankDescription}
+                </DescriptionText>
+                <div className={styles.bankDetails}>
+                    <BlockTextOutput
+                        label={strings.nationalSocietyBankName}
+                        value=""
+                        invalidText
+                    />
+                    <BlockTextOutput
+                        label={strings.nationalSocietyBankAccountNumber}
+                        value=""
+                        invalidText
+                    />
+                    <BlockTextOutput
+                        label={strings.nationalSocietySwiftCode}
+                        value=""
+                        invalidText
+                    />
+                    <BlockTextOutput
+                        label={strings.nationalSocietyAmount}
+                        value={strings.nationalSocietyAmountCHF}
+                    />
+                    <BlockTextOutput
+                        label={strings.nationalSocietyAdvancePayment}
+                        value=""
+                        invalidText
+                    />
+                </div>
```

Comment:
This doesn't seem to be right, the props `value` and `invalidText` doesn't seem to be used as intended. We should create separate component according to it's purpose.